processor/redactionprocessor/processor.go

```diff
@@ -43,6 +45,11 @@ func newRedaction(ctx context.Context, config *Config, logger *zap.Logger) (*red
 		// TODO: Placeholder for an error metric in the next PR
 		return nil, fmt.Errorf("failed to process block list: %w", err)
 	}
+	blockKeysRegexList, err := makeRegexList(ctx, config.BlockedKeyPatterns)
+	if err != nil {
+		// TODO: Placeholder for an error metric in the next PR
```

Comment:
Can we have an issue for any TODO in the code?

Reply:
this TODO comment was copied from a block right above this. I have zero context what needs to be done here, but added it also here, since it's present in every block that is creating regex list

Reply:
Okay, fair, feel free to file an issue to investigate the mysterious TODOs then, but no need for it to block merging this PR :)